src/util/generatePreview.js

```diff
@@ -39,6 +39,63 @@ const errorHandlerScript = `(${function() {
   };
 }.toString()}());`;
 
+const elementHighlighterScript = `(${function() {
+  let selector = '';
```

Comment:
Why declare this in the outer semi-global scope? Can’t we just assign it to a local inside the event listener and then pass it as an argument to `generateCovers`?

Reply:
This is to store the selector so the scroll and resizes listeners can update with the given selector.

Reply:
Sure, but we can accomplish the same thing by declaring a `const selector = selectorFromMessage` in the event listener, and then passing it as an argument to `generateCovers`, right?